src/PolygonDraw/validators.ts

```diff
@@ -1,8 +1,4 @@
 import { Coordinate } from 'types';
 import { isPolygonClosed } from '../helpers';
 
-export const isValidPolygon = (polygon: Coordinate[]) => (
-    polygon &&
-    polygon.length > 3 &&
-    isPolygonClosed(polygon)
-);
+export const isValidPolygon = (polygon: Coordinate[]) => polygon && polygon.length > 3 && isPolygonClosed(polygon);
```

Comment:
improvement: could we please check if isClosedPolygon is definitely working fine in all cases from an implementation perspective, I got a bit confused when I saw the logic in that method.

Reply:
I guess it's still confusing for me. What's the use case for closing a polygon which is invalid?